README.md

```diff
@@ -39,23 +39,33 @@ help
 `plotsbot-ryzokuken help [<module>...]`: Prints out this descriptive help message for each mentioned module. If no modules are specified, print the help message for ALL modules.`chatbot` is not the name of a valid module. Try looking up the `chatbot` module instead.
 ```
 
-## Services
+## Behaviors
 
-Services are different from interfaces as they carry out computational tasks and not I/O tasks like interfaces do. Therefore, the bot takes input from an interface, directs it to a service which would preform actions on the data and return data which the bot will output through another interface.
+Behaviors are clearly defined tasks performed by the bot. A behavior may use any third-party functions to perform this action, but it needs to expose a few mandatory fields that are used by the chatbot to trigger the behavior at the appropriate time.
 
-### Chatbot
+#### Behavior Specification
```

Comment:
Should I make another subheading called `Behavior Model` below the original `Behavior` subheading and add this in there?